src/main/java/ru/itmo/roguelike/characters/Actor.java

```diff
@@ -26,6 +26,17 @@
 
     protected float radius;
     protected Mover mover = new Mover();
+    /**
+     * Resistance of additional armor in percent
+     * Getting mob_damage*def*armorResistance points of damage
+     */
+    protected double armorResistance = 1;
```

Comment:
Why is it field of actor? This value should be taken from the armor instance that currently used